user_docs/en/changes.t2t

```diff
@@ -29,10 +29,10 @@ What's New in NVDA
 
 == Changes for Developers ==
 - Note: this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated.
-- `LiveText._getTextLines` has been removed. (#11639)
-  - Instead, override `_getText` which returns a string of all text in the object.
-- `LiveText` objects can now calculate diffs by character. (#11639)
-  - To alter the diff behaviour for some object, override the `diffAlgo` property (see the docstring for details).
+- <code>LiveText._getTextLines</code> has been removed. (#11639)
+  - Instead, override <code>_getText</code> which returns a string of all text in the object.
+- <code>LiveText</code> objects can now calculate diffs by character. (#11639)
+  - To alter the diff behaviour for some object, override the <code>diffAlgo</code> property (see the docstring for details).

```

Comment:
I'm not sure about using `<code>` directly. txt2tags does support several options eg two times back tick for monospace, two times double quote for raw and two times single quote for tagged. I'm not really sure what raw and tagged mean:

Here is a quick overview of these formatting options: https://txt2tags.org/manpage.html
The [user guide](https://txt2tags.org/userguide/) has more options and info in "Part V - Mastering Marks"